packages/atlas/src/views/studio/VideoWorkspace/VideoForm/utils.ts

```diff
@@ -1,7 +1,11 @@
 import { VideoFormData, VideoWorkspaceVideoFormFields } from '@/providers/videoWorkspace'
 
+type Public = {
+  isPublic?: boolean
+}
+
 export const convertVideoFormDataToFormFields = (
-  videoFormData: VideoFormData
+  videoFormData: Omit<VideoFormData, 'isPublic'> & Public
```

Comment:
What exactly is happening here?

1. `isPublic` property doesn't exists on `VideoFormData` so no point in excluding it
2. You add `Public` to the type but it's never used - when `videoFormData` is destructured, `isPublic` from `metadata` is used

Seems this change isn't really needed

Reply:
Nevermind, you are right, I don't what happend with my IDE yesterday, I had some errors related to `videoFormData` types.